addons/point_of_sale/static/src/app/store/pos_store.js

```diff
@@ -1083,6 +1085,7 @@ export class PosStore extends Reactive {
         if (this.isOpenOrderShareable() || removeFromServer) {
             if (typeof order.id === "number" && !order.finalized) {
                 this.addPendingOrder([order.id], true);
+                this.syncAllOrdersDebounced();
```

Comment:
why debounced ? i think you can directly call syncallorders

Reply:
@davidmonnom If you proceed with removing all the orders at once (e.g., when closing a session with open orders), it will trigger multiple calls unnecessarily, whereas only one call is needed.